### ISSUE TITLE
Wrong variable name at kanji_content_detect.js:24?

### ISSUE THREAD

**Issue by Fyren** [kuanyui/Furiganaize#4]
`mutationObserver` looks like it should be `observer` here, but I didn't put any effort into it besides looking up the line of code after seeing the error in my browser console:

https://github.com/kuanyui/Furiganaize/blob/0b1035bea184f0538988be01dbfc4894f59e4594/kanji_content_detect.js#L23-L24

**Comment by kuanyui** [comment]
Fixed

**Comment by mngyuan** [comment]
@kuanyui, was this fix included in 0.5.1? Line 24 is still causing 
```
TypeError: MutationObserver.observe: One of 'childList', 'attributes', 'characterData' must not be false.
```
on every page I visit, and I have 0.5.1 according to Firefox's add-ons page.

**Comment by kuanyui** [comment]
Fixed.
